mlir/test/Examples/transform/ChH/full.mlir

```diff
@@ -19,6 +19,7 @@
 // tensors annotated with attributes from the `bufferization` dialect. These
 // attributes hint the bufferization pass to assume buffers can be directly
 // used for these tensors without reshaping.
+module @payload attributes { transform.target_tag = "payload" } {
 func.func @conv(
     %input: !tinput {bufferization.writable = false,
                      bufferization.access = "read",
```

Comment:
Hi @matthias-springer , we are trying to use `debug-payload-root-tag=payload` to limit the `moduleOP bufferization` to the payload module. (Or it will trigger an error for no `ReturnLike`.)

It seems that the added payload module in this test blocks some passes after `transform-interpreter`.  The checks fail since the IRs like `llvm.intr.fma` are not produced. Maybe there are some integration issues? Is there any better way to exclude `transform.named_sequence` from bufferziation? Thank you very much.
 
https://github.com/llvm/llvm-project/blob/828783177f71d95522763fba92ef1e42dc6101c7/mlir/test/Examples/transform/ChH/full.mlir#L379-L388

Reply:
I forgot how exactly `transform.target_tag = "payload"` works. Maybe the transform script is only applied to the contents of the module but not the module itself. Can you try adding a nested `module`?

Reply:
Thank you. I have tried adding a nested module but we still got the same error. We will try exploring the possible fix.

Reply:
Hi @tzunghanjuang 

can you try the following patch

```patch
diff --git a/mlir/test/Examples/transform/ChH/full.mlir b/mlir/test/Examples/transform/ChH/full.mlir
index 005ac5a9ba..190d9672fd 100644
--- a/mlir/test/Examples/transform/ChH/full.mlir
+++ b/mlir/test/Examples/transform/ChH/full.mlir
@@ -1,8 +1,6 @@
 // RUN: mlir-opt %s --transform-interpreter="debug-payload-root-tag=payload" \
-// RUN:             --test-transform-dialect-erase-schedule \
-// RUN:             --math-uplift-to-fma \
-// RUN:             --convert-bufferization-to-memref \
-// RUN:             --test-lower-to-llvm |\
+// RUN:             --test-transform-dialect-erase-schedule \|
+// RUN: mlir-opt -pass-pipeline='builtin.module(builtin.module(math-uplift-to-fma,convert-bufferization-to-memref,test-lower-to-llvm))' - \|
 // RUN: FileCheck %s
 
 // Fixed-size tensor types to be used in convolution.
 ```
 
 I think the issue is not from the payload module not being transformed correctly by the transform schedule, but by the other passes not being schedule properly. So I created a pass pipeline that acts on nested modules. I tested this locally and it works.

Reply:
Thank you @erick-xanadu. It also works on my side. I have pushed the patch to see what will happen on the CI.